### PR TITLE
Add 2021q2 entry for libglvnd

### PR DESCRIPTION
**Pull request by kev009**

### CODE REVIEW

File: 2021q2/libglvnd.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+=== libglvnd on FreeBSD
+
+Contact: Kevin Bowling <kbowling@FreeBSD.org>
+
+Links: +
+link:https://gitlab.freedesktop.org/glvnd/libglvnd[libglvnd] URL: link:https://gitlab.freedesktop.org/glvnd/libglvnd[https://gitlab.freedesktop.org/glvnd/libglvnd]
+
+libglvnd (GL Vendor-Neutral Dispatch) "is a vendor-neutral dispatch layer for arbitrating OpenGL API calls
+between multiple vendors. It allows multiple drivers from different vendors to
+coexist on the same filesystem, and determines which vendor to dispatch each
```
Comment (pauamma) [inline 2021q2/libglvnd.adoc:10]: "in the same filesystem" maybe?

Also, I'd make it "which vendor's".

Comment (mattst88) [reply 2021q2/libglvnd.adoc:10]: > "in the same filesystem" maybe?

It's part of a quote from the project's README: https://gitlab.freedesktop.org/glvnd/libglvnd/-/blob/master/README.md

> Also, I'd make it "which vendor's".

Part of the same quote, but in any case `'s` doesn't make sense to me here. The original sentence is a bit complex but looks correct to me.

File: 2021q2/libglvnd.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+API call to at runtime.  Both GLX and EGL are supported, in any combination with OpenGL and OpenGL ES."
+
+Making this the default GL implementation in FreeBSD Ports unblocks a number of efforts and aligns us with future graphics stacks.
+
+It cleans up the ability to have multiple GL implementations on one system or image, regardless of what hardware is in use, by eliminating the need for one-or-the-other hard links for these libraries.  This can in turn be used by Nvidia Optimus setups, or using your fancy PCIe interconnect to have multiple vendor GPUs for any reason.  It will support future mesa releases where a concurrent LTS (Long Term Support) branch install can be used to fill in drivers that will be removed from newer mesa releases.
```
Comment (mattst88) [inline 2021q2/libglvnd.adoc:15]: I'd capitalize `Mesa` (occurs twice in this line).

File: 2021q2/libglvnd.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Software like KDE and Firefox can use EGL contexts under X11 (with OpenGL) and Wayland, and in the case of upcoming Firefox releases it may match the speed of Google Chrome when doing so.
+
+The library uses various implementation details to minimize any performance impact from indirection, with platform specific support for aarch64, amd64, armv7, i386, and powerpc64 (ELFv2).
+
+Finally, it helps reduces over-linking with a new libOpenGL to provide OpenGL without X11 for headless servers or kms or Wayland environments.
+
+If there is any remaining fallout from this change please add kbowling@ to the PR watchers.
+
+Thank you to Jan Beich <jbeich@FreeBSD.org> for doing the ports infrastructure work and Matt Turner (Gentoo/freedesktop) for helping me understand the architecture and how to handle this change in a source-based distribution, as well as merging a change in libGLU upstream to support building without X11 dependency.
```
Comment (mattst88) [inline 2021q2/libglvnd.adoc:25]: Glad to help!

Contents are accurate as far as I can tell, and it reads well to me.

**Pull request closed without merging**